edge-apps/clock/static/css/common.css

```diff
@@ -0,0 +1,329 @@
+/* stylelint-disable media-feature-range-notation */
+
+/* Common Styles */
+
+/* Global Fonts */
+@font-face {
+  font-family: Aeonik;
+  src: url('../font/Aeonik-Regular.woff2') format('woff2'),
+    url('../font/Aeonik-Regular.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+
+/* Typography */
+html {
+  font-size: 14px;
+}
+
+body {
+  font-family: Aeonik, sans-serif;
+  height: 100vh;
+}
+
+/* Colors (Theme) */
+:root {
+  --theme-color-primary: #972EFF;
+  --theme-color-secondary: #ADAFBE;
+  --theme-color-tertiary: #FFF;
+  --theme-color-background: #C9CDD0;
+}
+
+/* Reset */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Components */
+
+
+.main-container {
+  display: flex;
+  flex-direction: row;
+  gap: 2rem;
+  height: 100%;
+  padding: 2rem;
+  background-color: var(--theme-color-background);
+}
+
+.primary-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 50%;
+  height: 100%;
+  border-radius: 3rem;
+  gap: 2rem;
+  line-height: 1;
+  background-color: var(--theme-color-primary);
+  color: var(--theme-color-tertiary);
+}
+
+
+/* Secondary Container */
+.secondary-container {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  justify-content: space-between;
+}
+
+.row-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: 2rem;
+  height: 50%;
+}
+
+
+.secondary-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: auto;
+  gap: 2rem;
+  border-radius: 3.481rem;
+  background-color: var(--theme-color-tertiary);
+}
+
+/* Media Query */
+
+@media (orientation: portrait) {
+  .main-container {
+    flex-direction: column;
+  }
+
+  .primary-card {
+    width: 100%;
+    height: 50%;
+  }
+
+  .secondary-container {
+    width: 100%;
+    height: 50%;
+  }
+}
+
+/* 480px 720px Clock Container need to fix */
+
+/* TODO: Fix the Clock Circle issue : When Scale to .5, circle change to oval */
```

Comment:
Not required probably

Reply:
It's have issue with 480px portrait and 720px portrait screen for now, the clock icon is not working as expected to me. So I just made it as todo and will fix it later.